test/core/PriceManager.js

```diff
@@ -25,44 +25,34 @@ describe("PriceManager", function () {
     let jpy
     let usdc
     let usdt
-    let btcPriceFeed
-    let ethPriceFeed
-    let dogePriceFeed
-    let gbpPriceFeed
-    let eurPriceFeed
-    let jpyPriceFeed
-    let usdcPriceFeed
-    let usdtPriceFeed
-    let vlpPriceFeed
-    let vaultPriceFeed
+    let btcPriceId
+    let ethPriceId
+    let dogePriceId
+    let gbpPriceId
+    let eurPriceId
+    let jpyPriceId
+    let usdcPriceId
+    let usdtPriceId
     let cooldownDuration
     let feeRewardBasisPoints // FeeRewardBasisPoints 70%
-
+    let pyth
     before(async function () {
         btc = await deployContract("BaseToken", ["Bitcoin", "BTC", 0])
-        btcPriceFeed = await deployContract("FastPriceFeed", [])
-    
+        btcPriceId = "0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43"
```

Comment:
please add a comment where to fetch these priceIds, linking to docs for easier verify